joomla/joomla-cms
# [4.0] Fix Untranslated tip

### Summary of Changes 

As title says. The tip displays in the articles Manager when an article is Unpublished.
It concerns the `COM_CONTENT_CHANGE_STAGE` lang string

### Testing Instructions
Create an article as Unpublished. 
make sure publish up and down are empty
Hover the Status icon.

### Before patch

<img width="652" alt="Screen Shot 2019-11-28 at 11 06 52" src="https://user-images.githubusercontent.com/869724/69797374-d8b71100-11cf-11ea-9270-7643b9b25833.png">


### After patch
<img width="618" alt="Screen Shot 2019-11-28 at 11 05 09" src="https://user-images.githubusercontent.com/869724/69797391-e2407900-11cf-11ea-9d25-cc0797899c81.png">

### About this patch
It is the simplest solution but we have another solution by passing through Text the string in both 
`articles/default.php` and `featured/default.php`

```
													->addState(ContentComponent::CONDITION_PUBLISHED, '', 'publish', 'COM_CONTENT_CHANGE_STAGE', ['tip_title' => 'JPUBLISHED'])
													->addState(ContentComponent::CONDITION_UNPUBLISHED, '', 'unpublish', 'COM_CONTENT_CHANGE_STAGE', ['tip_title' => 'JUNPUBLISHED'])
													->addState(ContentComponent::CONDITION_ARCHIVED, '', 'archive', 'COM_CONTENT_CHANGE_STAGE', ['tip_title' => 'JARCHIVED'])
													->addState(ContentComponent::CONDITION_TRASHED, '', 'trash', 'COM_CONTENT_CHANGE_STAGE', ['tip_title' => 'JTRASHED'])
													
```
I can change the PR if judged necessary.
